src/controllers/EvidenceDownloadController.ts

```diff
@@ -13,16 +18,23 @@ import { DOWNLOAD_FILE_PAGE_URI } from 'app/utils/Paths';
 const template = 'download-file';
 const errorCustomTemplate = 'error-custom';
 
-@controller(DOWNLOAD_FILE_PAGE_URI, FileTransferFeatureMiddleware)
+@controller(DOWNLOAD_FILE_PAGE_URI,
+    SessionMiddleware,
+    FileRestrictionsAuthMiddleware,
```

Comment:
Let's say we have disabled the file transfer feature, would the Auth Middleware specific to File Restrictions cause any issues?

Reply:
The user would log in as usual but then it will just kick you to the penalty details screen